Fixing test case for get_location_string method.

Squirrel18 opened the pull request:
## Description:

This PR fixes the test case made it for get_location_string method in favor of this [OSPR](https://github.com/eduNEXT/flow-control-xblock/pull/25).

## Test:

 - Activate the virtual environment python.
 - Run the **tox** command.

## Reviewers:

- [ ] @felipemontoya 
- [ ] @diegomillan

felipemontoya commented on flow_control/tests/test_flowcontrol.py:
```diff
@@ -98,29 +99,38 @@ def test_get_location_string(self):
+        # We want to pass arbitrary times for random.choice.
+        for index in range(5):  # pylint: disable=unused-variable
+            self.block.course_id = MagicMock()
+            self.block.course_id.BLOCK_PREFIX = random.choice(course_strings)
```
Why are you making this random? is it not enough to go through all the course_strings one by one?

Squirrel18 replied:
This random is just for get different strings for BLOCK_PREFIX and BLOCK_TYPE_PREFIX. 
Could be made it in other way? @felipemontoya

felipemontoya commented on flow_control/tests/test_flowcontrol.py:
```diff
@@ -98,29 +99,38 @@ def test_get_location_string(self):
+            'course',
+            'course-v1',
+            'Course',
+            'course+course',
```
Do you think this should be a valid string? or are you expecting the tests to fail when they reach this value?

Squirrel18 replied:
It is just for fail in case. @felipemontoya

Squirrel18 commented:
One more review please. @felipemontoya @diegomillan

Squirrel18 committed:
Fixing test case for get_location_string method.
Trailing comma.
Cleaning code.
Making changes on test.
Remove unused inline comment.
Change test case.

Squirrel18 commented:
@felipemontoya @diegomillan Do you think it is ready to be merged?
